server/api.js

```diff
@@ -24,20 +24,27 @@ router.get("/bookings", async (req, res) => {
 
 router.post("/bookings", async function (req, res) {
 	const userName = req.body.name;
+	const deskId = req.body.deskId;
```

Comment:
desk_id, probably for consistency with the other filed names, and with the objects returned by the GET request (sorry, naming things well is hard, but worth doing :) )